R/estimateDiversity.R

```diff
@@ -118,8 +127,8 @@
 #'
 #' # It is recommended to specify also the final names used in the output.
 #' se <- estimateDiversity(se,
-#'   index = c("shannon", "gini_simpson", "inverse_simpson", "coverage", "fisher"),
-#'    name = c("Shannon", "GiniSimpson",  "InverseSimpson",  "Coverage", "Fisher"))
+#'   index = c("shannon", "gini_simpson", "inverse_simpson", "coverage", "fisher", "pd"),
+#'    name = c("Shannon", "GiniSimpson",  "InverseSimpson",  "Coverage", "Fisher", "Pd"))
```

Comment:
I would propose naming this faith/Faith - more specific than "pd". There can also be other "phylogenetic diversity" measures (now or in the future).

Reply:
Changed it to faith